packages/webapp/pages/onboarding.tsx

```diff
@@ -143,6 +150,10 @@ export function OnboardPage(): ReactElement {
     feature: featureOnboardingSources,
     shouldEvaluate: shouldEnrollSourceSelection,
   });
+  const appExperiment = useConditionalFeature({
+    feature: feature.onboardingAndroid,
+    shouldEvaluate: checkIsBrowser(UserAgent.Android),
```

Comment:
We should also add the trigger that it should only evaluate if android browser + closer to the step, basically you can add the evaluation we use for source onboarding for example.